view/theme/frio/js/theme.js

```diff
@@ -941,4 +942,13 @@ function toggleDropdownText(elm) {
 function hasClass(elem, cls) {
 	return (" " + elem.className + " ").indexOf(" " + cls + " ") > -1;
 }
+
+// Go to parent for "is answer to <user>" link in comments below username
+function goToParentItem(plink, pguid) {
+	history.pushState({},"",plink);
+	var loc = plink.split("/");
+	loc.pop();
+	loc.push(pguid);
+	location.href = loc.join("/");
+}
```

Comment:
There's already a `scrollToItem()` function in the same file that you can use for this.

Here's the usage: `scrollToItem("item-" + itemGuid)`.